### DISCUSSION TITLE
I want to create a bot when telegram.Bot(token) passes an http proxy 127.0.0.1:7890,bot

### DISCUSSION THREAD

**Discussion by 2622594863** [python-telegram-bot/python-telegram-bot#4097] [Q&A]
I want to create a bot when telegram.Bot(token) passes an http proxy 127.0.0.1:7890,bot

**Comment by Bibo-Joshi** [comment]
https://github.com/python-telegram-bot/python-telegram-bot/wiki/Ask-Right
